COURSE.md

```diff
@@ -1,38 +1,61 @@
 # Course outline
 
-### Low-level goals/topics
 This course is divided into separate modules, detailed below. Module A1-A4 are introductory, and contain what's needed to learn about the contents of the other modules. Therefore, you're advised to teach modules A1-A4 at the start of the course, but you can pick and mix the other modules.
 
-#### 0 - Course introduction
+## Learning objectives
+At the end of the course, the student is able to:
+- Describe the problems Rust aims to solve, and how (A1)
```

Comment:
and why! Why do we need a new language to solve these problems?

Reply:
I agree!